### PR TITLE
Refactor control-panel div from App into a WorkspaceControlPanel component.

### PR DESCRIPTION
**Pull request by jkeck**
Rationale being that issues like #1586 and #1587 will be able to build on top of this and not have to all work in the top-level App component.

🤷‍♂️

### CODE REVIEW

**Comment by jkeck** [comment]
Pushed up fix to the conflict.  Also, I had added a test last night which allowed the build to pass using the existing CC config (w/ 84% diff coverage), just as a heads up.

File: src/components/WorkspaceControlPanel.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    super(props);
+    this.state = {
+      lastRequested: '',
+    };
+
+    this.setLastRequested = this.setLastRequested.bind(this);
+  }
+
+  /**
+   * setLastRequested - Sets the state lastRequested
```
Comment (jbaiter) [inline src/components/WorkspaceControlPanel.js]: I think that docstring could be a bit more descriptive, what exactly is `lastRequested`?

Comment (aeschylus) [reply src/components/WorkspaceControlPanel.js]: Originally this was a flag to allow us to display only the information for the most recently requested manifest. We can probably just delete this at this point, as the side panel designs don't really require this behaviour.

Comment (jkeck) [reply src/components/WorkspaceControlPanel.js]: I agree @jbaiter.  I was simply moving this comment w/ its accompanying code from another place to here in my component extraction, so not really sure I'm the best person to describe what the intent of the original author was.

**Pull request merged**